### DISCUSSION TITLE
Taro.getSetting 文档和TS定义缺少获取用户订阅消息的订阅状态

### DISCUSSION THREAD

**Discussion by theinkoftheink** [NervJS/taro#11453] [Q & A]
![image](https://user-images.githubusercontent.com/65798802/158299418-8e64874f-4f54-4c76-ad21-76679f128801.png)

![image](https://user-images.githubusercontent.com/65798802/158321717-c43da22c-a6fd-4fdf-a0fc-0e4d72876dc2.png)

**Comment by ZakaryCode** [comment]
欢迎 PR
